app/services/proxy_status/puppetca.rb

```diff
@@ -1,5 +1,5 @@
 module ProxyStatus
-  class PuppetCA < Base
+  class Puppetca < Base
```

Comment:
In https://github.com/theforeman/foreman/pull/9328 I chose to make CA an acronym and name the file `puppet_ca` instead. Didn't notice this PR until now.